### PR TITLE
Multiple code improvements - common-java:DuplicatedBlocks, squid:S1118, squid:S1854, squid:S1192, squid:RedundantThrowsDeclarationCheck, squid:S134, squid:SwitchLastCaseIsDefaultCheck, squid:S1213, squid:S1301, squid:S00116, squid:S1170, squid:S1488

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrences of Sonar rules
common-java:DuplicatedBlocks - Standard outputs should not be used directly to log anything.
squid:S1118 - Standard outputs should not be used directly to log anything.
squid:S1854 - Standard outputs should not be used directly to log anything.
squid:S1192 - Standard outputs should not be used directly to log anything.
squid:RedundantThrowsDeclarationCheck - Standard outputs should not be used directly to log anything.
squid:S134 - Standard outputs should not be used directly to log anything.
squid:SwitchLastCaseIsDefaultCheck - Standard outputs should not be used directly to log anything.
squid:S1213 - Standard outputs should not be used directly to log anything.
squid:S1301 - Standard outputs should not be used directly to log anything.
squid:S00116 - Standard outputs should not be used directly to log anything.
squid:S1170 - Standard outputs should not be used directly to log anything.
squid:S1488 - Standard outputs should not be used directly to log anything.
You can find more information about the issue here:
https://dev.eclipse.org/sonar/rules/show/common-java:DuplicatedBlocks
https://dev.eclipse.org/sonar/rules/show/squid:S1118
https://dev.eclipse.org/sonar/rules/show/squid:S1854
https://dev.eclipse.org/sonar/rules/show/squid:S1192
https://dev.eclipse.org/sonar/rules/show/squid:RedundantThrowsDeclarationCheck
https://dev.eclipse.org/sonar/rules/show/squid:S134
https://dev.eclipse.org/sonar/rules/show/squid:SwitchLastCaseIsDefaultCheck
https://dev.eclipse.org/sonar/rules/show/squid:S1213
https://dev.eclipse.org/sonar/rules/show/squid:S1301
https://dev.eclipse.org/sonar/rules/show/squid:S00116
https://dev.eclipse.org/sonar/rules/show/squid:S1170
https://dev.eclipse.org/sonar/rules/show/squid:S1488
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

File: cucumber-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/cucumber/helpers/Helpers.java [hunk 1]
```diff
@@ -167,7 +167,18 @@ public CharSequence apply(List<Row> rows, Options arg1) throws IOException {
...
                 return tableContent;
             }
-        });
-        return handlebar;
+        };
+    }
+
+    private CharSequence checkState(String arg0, String retValue1, String retValue2, String retValue3) {
+        switch (arg0.toLowerCase()) {
+            case "skipped":
```
Comment (BogdanLivadariu) [inline cucumber-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/cucumber/helpers/Helpers.java]: I know this was already in, but do you think we can have these added to `Constants` and replace other occurrences ?

**Comment by BogdanLivadariu** [comment]
@georgekankava  do you think we can add some sonar reporting in ?

**Comment by georgekankava** [comment]
@BogdanLivadariu No I cannot work on that I am sorry, but you can if you would like.

File: cucumber-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/cucumber/helpers/Helpers.java (hunk 1 above)
Comment (georgekankava) [inline cucumber-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/cucumber/helpers/Helpers.java]: @BogdanLivadariu I will take a look.

**Comment by georgekankava** [comment]
@BogdanLivadariu Changes applied.

**Comment by BogdanLivadariu** [comment]
👍

**Comment by georgekankava** [comment]
@BogdanLivadariu Did you also want to move 'handlebar' keys in Constants in 'Helpers' class?

**Comment by BogdanLivadariu** [comment]
@georgekankava not sure if there will be other occurrences with their use, for the time being they may live there :)

**Pull request merged**